test/playwright/e2e/mobile-menu.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { test, expect } from '@playwright/test'
+
+import {
+  closeFilters,
+  closeMobileMenu,
+  enableNewHeader,
+  goToSearchTerm,
+  isMobileMenuOpen,
+  openContentTypes,
+  openFilters,
+} from '~~/test/playwright/utils/navigation'
+
+const mockUaString =
+  'Mozilla/5.0 (Android 7.0; Mobile; rv:54.0) Gecko/54.0 Firefox/54.0'
+const mobileFixture = {
+  viewport: { width: 640, height: 700 },
+  userAgent: mockUaString,
+}
```

Comment:
This duplicates part of the breakpoints utilities which already configure the user agent string and viewport. What is the reason to duplicate that here? Could we import the UA string from `breakpoints` at least to deduplicate there if the rest is not possible?